index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Assignment 10</title>
+        <meta charset="utf-8">
+        <link rel='stylesheet' href="styles.css">
+        <script src="ngonoessono_a10.js"></script>
+    </head>
+    <body>
+        <h1>Assignment 10</h1>
+        <img id="photo" src="01.jpg" />

```

Comment:
you didn't include 01.jpg and 02.jpg in your repo. so this img tag doesn't work.